test_integration.py fails on Windows: OSError Invalid argument (invalid file name)

### Summary
test_integration.py fail with invalid filename on Windows.

The file name is generated by:
https://github.com/ME-ICA/tedana/blob/c1423c16235b872b0c8a4b3428dba4c1a99b9f58/tedana/workflows/tedana.py#L333 which creates two `:`s (the symbol : colon) inside datetime, which is invalid on Windows.
```
E       OSError: [Errno 22] Invalid argument: 'C:\\tmp\\data\\five-echo\\TED.five-echo\\tedana_2020-02-28T16:39:04.tsv'
```

### Additional Detail
```
(base) C:\Users\KUAN\tedana>pytest tedana
============================= test session starts =============================
platform win32 -- Python 3.7.0, pytest-3.8.0, py-1.8.0, pluggy-0.7.1
rootdir: C:\Users\KUAN\tedana, inifile:
plugins: remotedata-0.3.0, openfiles-0.3.0, doctestplus-0.1.3, arraydiff-0.2
collected 50 items

tedana\tests\test_combine.py ...                                         [  6%]
tedana\tests\test_decay.py ......                                        [ 18%]
tedana\tests\test_gscontrol.py .                                         [ 20%]
tedana\tests\test_integration.py FFF                                     [ 26%]
tedana\tests\test_io.py ........                                         [ 42%]
tedana\tests\test_mapca.py ..                                            [ 46%]
tedana\tests\test_model_fit_dependence_metrics.py .                      [ 48%]
tedana\tests\test_model_kundu_metrics.py .                               [ 50%]
tedana\tests\test_selection.py .                                         [ 52%]
tedana\tests\test_stats.py .......                                       [ 66%]
tedana\tests\test_t2smap.py ....                                         [ 74%]
tedana\tests\test_utils.py ............                                  [ 98%]
tedana\tests\test_viz.py .                                               [100%]

================================== FAILURES ===================================
_________________________ test_integration_five_echo __________________________

skip_integration = False

    def test_integration_five_echo(skip_integration):
        """ Integration test of the full tedana workflow using five-echo test data
        """

        if skip_integration:
            pytest.skip('Skipping five-echo integration test')
        out_dir = '/tmp/data/five-echo/TED.five-echo'
        if os.path.exists(out_dir):
            shutil.rmtree(out_dir)

        # download data and run the test
        download_test_data('https://osf.io/9c42e/download',
                           os.path.dirname(out_dir))
        prepend = '/tmp/data/five-echo/p06.SBJ01_S09_Task11_e'
        suffix = '.sm.nii.gz'
        datalist = [prepend + str(i + 1) + suffix for i in range(5)]
        tedana_workflow(
            data=datalist,
            tes=[15.4, 29.7, 44.0, 58.3, 72.6],
            out_dir=out_dir,
            tedpca='aic',
            fittype='curvefit',
            tedort=True,
>           verbose=True)

tedana\tests\test_integration.py:96:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
tedana\workflows\tedana.py:343: in tedana_workflow
    log_handler = logging.FileHandler(logname)
..\Anaconda3\lib\logging\__init__.py:1041: in __init__
    StreamHandler.__init__(self, self._open())
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <[AttributeError("'FileHandler' object has no attribute 'level'") raised in repr()] FileHandler object at 0x139aa0a9780>

    def _open(self):
        """
            Open the current base file with the (original) mode and encoding.
            Return the resulting stream.
            """
>       return open(self.baseFilename, self.mode, encoding=self.encoding)
E       OSError: [Errno 22] Invalid argument: 'C:\\tmp\\data\\five-echo\\TED.five-echo\\tedana_2020-02-28T16:39:04.tsv'

..\Anaconda3\lib\logging\__init__.py:1070: OSError
_________________________ test_integration_four_echo __________________________

skip_integration = False

    def test_integration_four_echo(skip_integration):
        """ Integration test of the full tedana workflow using four-echo test data
        """

        if skip_integration:
            pytest.skip('Skipping four-echo integration test')
        out_dir = '/tmp/data/four-echo/TED.four-echo'
        if os.path.exists(out_dir):
            shutil.rmtree(out_dir)

        # download data and run the test
        download_test_data('https://osf.io/gnj73/download',
                           os.path.dirname(out_dir))
        prepend = '/tmp/data/four-echo/'
        prepend += 'sub-PILOT_ses-01_task-localizerDetection_run-01_echo-'
        suffix = '_space-sbref_desc-preproc_bold+orig.HEAD'
        datalist = [prepend + str(i + 1) + suffix for i in range(4)]
        tedana_workflow(
            data=datalist,
            tes=[11.8, 28.04, 44.28, 60.52],
            out_dir=out_dir,
            tedpca='kundu-stabilize',
            gscontrol=['gsr', 't1c'],
            png_cmap='bone',
            debug=True,
>           verbose=True)

tedana\tests\test_integration.py:148:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
tedana\workflows\tedana.py:343: in tedana_workflow
    log_handler = logging.FileHandler(logname)
..\Anaconda3\lib\logging\__init__.py:1041: in __init__
    StreamHandler.__init__(self, self._open())
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <[AttributeError("'FileHandler' object has no attribute 'level'") raised in repr()] FileHandler object at 0x139aa0664a8>

    def _open(self):
        """
            Open the current base file with the (original) mode and encoding.
            Return the resulting stream.
            """
>       return open(self.baseFilename, self.mode, encoding=self.encoding)
E       OSError: [Errno 22] Invalid argument: 'C:\\tmp\\data\\four-echo\\TED.four-echo\\tedana_2020-02-28T16:39:20.tsv'

..\Anaconda3\lib\logging\__init__.py:1070: OSError
_________________________ test_integration_three_echo _________________________

skip_integration = False

    def test_integration_three_echo(skip_integration):
        """ Integration test of the full tedana workflow using three-echo test data
        """

        if skip_integration:
            pytest.skip('Skipping three-echo integration test')
        out_dir = '/tmp/data/three-echo/TED.three-echo'
        out_dir2 = '/tmp/data/three-echo/TED.three-echo-rerun'
        if os.path.exists(out_dir):
            shutil.rmtree(out_dir)

        # download data and run the test
        download_test_data('https://osf.io/rqhfc/download',
                           os.path.dirname(out_dir))
        tedana_workflow(
            data='/tmp/data/three-echo/three_echo_Cornell_zcat.nii.gz',
            tes=[14.5, 38.5, 62.5],
            out_dir=out_dir,
            low_mem=True,
>           tedpca='mdl')

tedana\tests\test_integration.py:174:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
tedana\workflows\tedana.py:343: in tedana_workflow
    log_handler = logging.FileHandler(logname)
..\Anaconda3\lib\logging\__init__.py:1041: in __init__
    StreamHandler.__init__(self, self._open())
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <[AttributeError("'FileHandler' object has no attribute 'level'") raised in repr()] FileHandler object at 0x139ab0d79e8>

    def _open(self):
        """
            Open the current base file with the (original) mode and encoding.
            Return the resulting stream.
            """
>       return open(self.baseFilename, self.mode, encoding=self.encoding)
E       OSError: [Errno 22] Invalid argument: 'C:\\tmp\\data\\three-echo\\TED.three-echo\\tedana_2020-02-28T16:39:27.tsv'

..\Anaconda3\lib\logging\__init__.py:1070: OSError
============================== warnings summary ===============================
C:\Users\KUAN\Anaconda3\lib\site-packages\h5py\_hl\base.py:19: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import (Mapping, MutableMapping, KeysView,
C:\Users\KUAN\Anaconda3\lib\site-packages\h5py\_hl\base.py:19: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import (Mapping, MutableMapping, KeysView,
C:\Users\KUAN\Anaconda3\lib\site-packages\h5py\_hl\base.py:19: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import (Mapping, MutableMapping, KeysView,
C:\Users\KUAN\Anaconda3\lib\site-packages\h5py\_hl\base.py:19: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import (Mapping, MutableMapping, KeysView,
C:\Users\KUAN\Anaconda3\lib\site-packages\h5py\_hl\base.py:19: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import (Mapping, MutableMapping, KeysView,

C:\Users\KUAN\Anaconda3\lib\site-packages\socks.py:58: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import Callable

C:\Users\KUAN\Anaconda3\lib\site-packages\pyparsing.py:3245: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  elif isinstance( exprs, collections.Iterable ):

C:\Users\KUAN\tedana\tedana\utils.py:35: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data = check_niimg(data).get_data()
C:\Users\KUAN\tedana\tedana\utils.py:35: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data = check_niimg(data).get_data()
C:\Users\KUAN\tedana\tedana\utils.py:35: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data = check_niimg(data).get_data()
C:\Users\KUAN\tedana\tedana\decay.py:163: RuntimeWarning: divide by zero encountered in true_divide
  t2s = 1. / betas[1, :].T

C:\Users\KUAN\tedana\tedana\decay.py:163: RuntimeWarning: divide by zero encountered in true_divide
  t2s = 1. / betas[1, :].T

C:\Users\KUAN\tedana\tedana\utils.py:35: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data = check_niimg(data).get_data()
C:\Users\KUAN\tedana\tedana\utils.py:35: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data = check_niimg(data).get_data()
C:\Users\KUAN\tedana\tedana\utils.py:35: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data = check_niimg(data).get_data()

C:\Users\KUAN\tedana\tedana\utils.py:35: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data = check_niimg(data).get_data()
C:\Users\KUAN\tedana\tedana\utils.py:35: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data = check_niimg(data).get_data()
C:\Users\KUAN\tedana\tedana\utils.py:35: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data = check_niimg(data).get_data()
C:\Users\KUAN\tedana\tedana\tests\test_io.py:40: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  assert np.allclose(ref.get_data(), nib.load(fnames[0]).get_data())
C:\Users\KUAN\tedana\tedana\tests\test_io.py:40: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  assert np.allclose(ref.get_data(), nib.load(fnames[0]).get_data())
C:\Users\KUAN\tedana\tedana\utils.py:37: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data = check_niimg(data).get_data()
C:\Users\KUAN\tedana\tedana\utils.py:37: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data = check_niimg(data).get_data()
C:\Users\KUAN\tedana\tedana\utils.py:37: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data = check_niimg(data).get_data()
C:\Users\KUAN\tedana\tedana\io.py:411: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  fdata = utils.load_image(img.get_data().reshape(nx, ny, nz, n_echos, -1, order='F'))

C:\Users\KUAN\tedana\tedana\io.py:411: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  fdata = utils.load_image(img.get_data().reshape(nx, ny, nz, n_echos, -1, order='F'))

C:\Users\KUAN\tedana\tedana\decomposition\ma_pca.py:474: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data_nib = data_nib.get_data()
C:\Users\KUAN\tedana\tedana\decomposition\ma_pca.py:475: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  mask_nib = mask_nib.get_data()
C:\Users\KUAN\tedana\tedana\decomposition\ma_pca.py:474: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data_nib = data_nib.get_data()
C:\Users\KUAN\tedana\tedana\decomposition\ma_pca.py:475: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  mask_nib = mask_nib.get_data()
C:\Users\KUAN\tedana\tedana\decomposition\ma_pca.py:474: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data_nib = data_nib.get_data()
C:\Users\KUAN\tedana\tedana\decomposition\ma_pca.py:475: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  mask_nib = mask_nib.get_data()

C:\Users\KUAN\tedana\tedana\utils.py:35: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data = check_niimg(data).get_data()
C:\Users\KUAN\tedana\tedana\utils.py:35: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data = check_niimg(data).get_data()
C:\Users\KUAN\tedana\tedana\utils.py:35: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data = check_niimg(data).get_data()
C:\Users\KUAN\tedana\tedana\decay.py:163: RuntimeWarning: divide by zero encountered in true_divide
  t2s = 1. / betas[1, :].T

C:\Users\KUAN\tedana\tedana\utils.py:35: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data = check_niimg(data).get_data()
C:\Users\KUAN\tedana\tedana\utils.py:35: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data = check_niimg(data).get_data()
C:\Users\KUAN\tedana\tedana\utils.py:35: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data = check_niimg(data).get_data()
C:\Users\KUAN\tedana\tedana\decay.py:163: RuntimeWarning: divide by zero encountered in true_divide
  t2s = 1. / betas[1, :].T

C:\Users\KUAN\tedana\tedana\utils.py:35: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data = check_niimg(data).get_data()
C:\Users\KUAN\tedana\tedana\utils.py:35: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data = check_niimg(data).get_data()
C:\Users\KUAN\tedana\tedana\utils.py:35: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data = check_niimg(data).get_data()
C:\Users\KUAN\tedana\tedana\decay.py:163: RuntimeWarning: divide by zero encountered in true_divide
  t2s = 1. / betas[1, :].T

C:\Users\KUAN\tedana\tedana\utils.py:35: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data = check_niimg(data).get_data()
C:\Users\KUAN\tedana\tedana\utils.py:35: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data = check_niimg(data).get_data()
C:\Users\KUAN\tedana\tedana\utils.py:35: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data = check_niimg(data).get_data()
C:\Users\KUAN\tedana\tedana\decay.py:163: RuntimeWarning: divide by zero encountered in true_divide
  t2s = 1. / betas[1, :].T

C:\Users\KUAN\tedana\tedana\utils.py:35: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data = check_niimg(data).get_data()
C:\Users\KUAN\tedana\tedana\utils.py:37: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data = check_niimg(data).get_data()
C:\Users\KUAN\tedana\tedana\tests\test_utils.py:77: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  assert utils.load_image(fimg.get_data()).shape == exp_shape

C:\Users\KUAN\tedana\tedana\utils.py:35: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data = check_niimg(data).get_data()
C:\Users\KUAN\tedana\tedana\utils.py:35: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data = check_niimg(data).get_data()
C:\Users\KUAN\tedana\tedana\utils.py:35: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data = check_niimg(data).get_data()
C:\Users\KUAN\tedana\tedana\utils.py:35: DeprecationWarning: get_data() is deprecated in favor of get_fdata(), which has a more predictable return type. To obtain get_data() behavior going forward, use numpy.asanyarray(img.dataobj).

* deprecated from version: 3.0
* Will raise <class 'nibabel.deprecator.ExpiredDeprecationError'> as of version: 5.0
  data = check_niimg(data).get_data()

-- Docs: https://docs.pytest.org/en/latest/warnings.html
============== 3 failed, 47 passed, 54 warnings in 47.55 seconds ==============
```
<!--
If desired, add suggested next steps.
If you foresee them in a particular order or priority, please use numbering
-->
### Next Steps

<!-- 
Thank you for submitting your issue!
If you do not receive a response within a calendar week, please post a comment on this issue to catch our attention. 
Some issues may not be resolved right away due to the volunteer nature of the project; thank you for your patience!
-->

--- a/tedana/tests/test_integration.py
+++ b/tedana/tests/test_integration.py
@@ -37,8 +37,8 @@ def check_integration_outputs(fname, outpath):
 
     # Checks for log file
     log_regex = ('^tedana_'
-                 '[12][0-9]{3}-[0-9]{2}-[0-9]{2}T[0-9]{2}:'
-                 '[0-9]{2}:[0-9]{2}.tsv$')
+                 '[12][0-9]{3}-[0-9]{2}-[0-9]{2}T[0-9]{2}'
+                 '[0-9]{2}[0-9]{2}.tsv$')
     logfiles = [out for out in existing if re.match(log_regex, out)]
     assert len(logfiles) == 1
 

--- a/tedana/workflows/tedana.py
+++ b/tedana/workflows/tedana.py
@@ -330,8 +330,8 @@ def tedana_workflow(data, tes, out_dir='.', mask=None,
     # create logfile name
     basename = 'tedana_'
     extension = 'tsv'
-    isotime = datetime.datetime.now().replace(microsecond=0).isoformat()
-    logname = op.join(out_dir, (basename + isotime + '.' + extension))
+    start_time = datetime.datetime.now().strftime('%Y-%m-%dT%H%M%S')
+    logname = op.join(out_dir, (basename + start_time + '.' + extension))
 
     # set logging format
     log_formatter = logging.Formatter(

